### ISSUE TITLE
Hard coded warning

### ISSUE THREAD

**Issue by ChuhanWang10** [cwchenwang/NeRF-SR#2]
Hi,

I tried to render the model by running "bash scripts/test_llff_downX.sh", then I got a hard-coded warning and the testing was interrupted like this:

![image](https://user-images.githubusercontent.com/80790361/181841484-d8cf7f8d-f07a-4159-9a05-2d6e02a05732.png)

I tried both the pre-trained model you shared and my own trained model. In both cases, the test did not work because of this warning. Can you tell me how to solve this? Thank you!

**Comment by ChuhanWang10** [comment]
Perhaps it is not about hard coding some parameters. It is weird because the execution is just stopped without any error occurring. Do you know how to solve this?  

And I am also curious about why the data_rgbs is hard coded as :
https://github.com/cwchenwang/NeRF-SR/blob/2d2d76e3c2abc8964620b02bc40fcea7aeac9585/models/nerf_downX_model.py#L331

Will not it change according to the size of data or resolution?

**Comment by cwchenwang** [comment]
We test one image for debugging and didn't clean up the code, see here at `nerf_downX_model.py`:
<img width="594" alt="image" src="https://user-images.githubusercontent.com/23579918/181867378-835a44fa-2bf0-49e9-a350-b9c16b7acb5d.png">
We have uploaded a new commit to solve this issue, thanks for pointing out.
Currently, the testing script for LLFF dataset is a virtual camera path we created (`--test_split test` in the script) for video demonstration, so it doesn't have ground truth rgbs. That's why we use a hard coded `self.data_rgbs`, which is just a placeholder for compatibility to previous code. As we only tested this on 1008x756 resolution, we didn't adapt to other resolution because of time, you are welcome to create a pull request to resolve this. In the paper, we report metrics on camera poses in the training set (`--test_split test_train` in the script).
